patch/u-boot/u-boot-rockchip/board_tinkerboard/3-4-rk3288-tinker-Enable-HDMI-output.patch.disabled

```diff
@@ -0,0 +1,176 @@
+From patchwork Thu Jan 23 14:12:21 2020
```

Comment:
Why is this patch added? If its needed, make sure to remove headers.

Reply:
After a second thought it was a mistake, I was a bit in a hurry when I did the PR and this come into. I will remove it from the PR ASAP. It is not needed at all, anyway it should bring working HDMI to u-boot for tinkerboard, but requires u-boot v2020.01 and probably it was much better to post this in the forum to discuss about. Sorry.